.github/workflows/daily.yml

```
@@ -896,7 +896,7 @@ jobs:
   build-macos:
     strategy:
       matrix:
-        os: [macos-11, macos-13]
+        os: [macos-12, macos-13, macos-14]
```

```suggestion
        os: [macos-12, macos-14]
```

We should probably build against 2 versions. Latest and the oldest supported as we were building it previously.